src/main/java/com/dap/DailyArtPrompt/controller/LoginController.java

```diff
@@ -0,0 +1,33 @@
+package com.dap.DailyArtPrompt.controller;
+
+import com.dap.DailyArtPrompt.model.UserResponse;
+import com.dap.DailyArtPrompt.service.LoginService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import java.util.Base64;
+
+@RestController
+@RequestMapping("/login")
+public class LoginController {
+    LoginService loginService;
+
+    public LoginController(LoginService loginService) {this.loginService = loginService;}
+
+    @PostMapping
+    public ResponseEntity<UserResponse> validateLogin(
+            @RequestHeader(value = "Authorization") String authorization
+    ) {
+        System.out.println("----------Login Controller: " + authorization);
```

Comment:
can re remove this sys out (or add an official `logger` if that's what you want)

Reply:
done